_guides/reverse-proxies.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: documentation
+title: TBD
+---
+
+Serving The Lounge through a reverse proxy instead of the built-in HTTP server
+comes with a few benefits:
+
+- The Lounge needs to run as root in order to serve it on port 80 (or 443 for
+  HTTPS), or you must always type the port number as part of the URL. Reverse
+  proxies usually abstract this, take control of ports 80 and 443, then redirect
+  traffic to The Lounge.
+
+- If the built-in HTTP server is already listening to port 80 or 443, this means
+  that your server cannot serve any other project through these ports. Reverse
+  proxies act as "gates" that route traffic from ports 80 and 443 to the
+  requested service.
+
+- While The Lounge comes with HTTPS support out of the box, any changes to the
+  HTTPS certificates (such as renewing them) requires a server restart. Using a
+  reverse proxy lets you reload the reverse proxy without having to restart The
+  Lounge.
+
+- When serving a website with HTTPS, it is common to also serve an HTTP
+  counterpart that redirects to the HTTPS version. This is not possible with the
+  built-in server that listens to a single port. It is usually trivial to do so
+  with reverse proxies.
+
+It however requires more configuration than just relying on the built-in server,
+so this guide helps going through the extra complexity.
+
+When using The Lounge behind a reverse proxy, set the `reverseProxy` option to
+`true` in your configuration file. This will instruct The Lounge to use the
+`X-Forwarded-For` header passed by your reverse proxy.
+
+This document assumes that your configuration of The Lounge binds to host
+`127.0.0.1` and port `9000`.
+
+## [Nginx](https://nginx.org/en/)
```

Comment:
At some point I added gzip configuration in the wiki, unsure if we want to copy it over?

https://github.com/thelounge/thelounge/wiki/Reverse-Proxies#nginx

Reply:
That's a good idea, done.